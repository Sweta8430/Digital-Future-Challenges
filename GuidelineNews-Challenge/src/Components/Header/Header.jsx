
const Header = () => {
    return (
        <header className="App-header">
            <div>
                <h1>WORLD NEWS TODAY</h1>
            </div>
        </header>
    );
};

export default Header;