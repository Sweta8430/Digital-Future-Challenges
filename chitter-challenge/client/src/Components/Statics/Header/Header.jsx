import 'bootstrap-icons/font/bootstrap-icons.css';
import '../../../style.css';
const Header = () => {
    return (
        <header>
            <nav className="navbar navbar-expand-lg ms-auto">
                <div className="container-fluid justify-content-start">
                    <a className = "headerh4" href="/">CHITTER</a> 
                </div>
        </nav>
        </header>
    );
};

export default Header;