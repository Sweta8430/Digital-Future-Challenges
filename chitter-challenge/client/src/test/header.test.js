import Header from "../Components/Statics/Header/Header";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";

describe("Header component test", () => {
  it("Header matched the snapshot", () => {
    const headerComponent = render(
      <MemoryRouter>
        <Header />
      </MemoryRouter>
    );
    expect(headerComponent).toMatchSnapshot();
  });
});
