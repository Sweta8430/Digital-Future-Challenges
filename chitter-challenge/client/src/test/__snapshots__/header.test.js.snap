// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component test Header matched the snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header>
        <nav
          class="navbar navbar-expand-lg ms-auto"
        >
          <div
            class="container-fluid justify-content-start"
          >
            <a
              class="headerh4"
              href="/"
            >
              CHITTER
            </a>
          </div>
        </nav>
      </header>
    </div>
  </body>,
  "container": <div>
    <header>
      <nav
        class="navbar navbar-expand-lg ms-auto"
      >
        <div
          class="container-fluid justify-content-start"
        >
          <a
            class="headerh4"
            href="/"
          >
            CHITTER
          </a>
        </div>
      </nav>
    </header>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
